easybuild/easyconfigs/n/numactl/numactl-2.0.10-GCC-4.9.2.eb

```diff
@@ -3,6 +3,8 @@ easyblock = 'ConfigureMake'
 name = 'numactl'
 version = '2.0.10'
 
+checksums = ['682c38305b2596967881f3d77bc3fc9c']
+
 homepage = 'http://oss.sgi.com/projects/libnuma/'
 description = """The numactl program allows you to run your application program on specific cpu's and memory nodes.
 It does this by supplying a NUMA memory policy to the operating system before running your program.
```

Comment:
while you're at it: leading space here, and the same for any other 'continuation' lines of the `description`